packages/table/src/cell/_cell.scss

```diff
@@ -21,6 +21,11 @@
     @include cell-content-align-vertical();
     color: transparent;
   }
+
+  &:focus {
+    // disable focus outline on cells; we already have the focus border
+    outline: none;
```

Comment:
does this rule affect accessibility? I'm not sure how accessibility features in browsers deal with focus style overrides. maybe it doesn't matter?

Reply:
This only affects visuals, not interactivity